packages/v4/src/content/design-guidelines/styles/typography/typography.md

```diff
@@ -22,132 +22,105 @@ We use Red Hat Display and Red Hat Text. Red Hat Display should be used for **la
 ## Usage guidelines
 Use typography to create visual hierarchy. A consistent and logical hierarchy makes it easier for users to quickly scan and understand information on a page.
 
+There are two components that can be used to place text content on a page. The [text component](/components/text) should be used for creating formatted blocks of text content and accepts all general HTML text formatting tags, including heading, paragraph, and list styles. The [title component](/components/title) is intended to be used specifically for headings or title text in cards or in similar elements. Title provides the flexibility to set the size of the text and heading level independently.
+
+The table below lists all common text styles that are used in PatternFly applications, how they should be used, style settings, and default heading levels, where applicable. Here are further details about the information contained in the table:
+
+* **Font family, font weight, line height, and font size:** These are the styles applied to that text category in the PatternFly CSS.
+
+* **Text style:** This is the name used to refer to that text style in the PatternFly design kit.
+
+* **Font size CSS variable:** This is the CSS variable that controls the size of this text. You can customize type sizes in your application by modifying the values associated with these variables. In general, this is not advised, and should only be done with caution, as changes to text style can have far reaching side effects on spacing within certain components.
+
+### Super hero heading (4xl)
 <TitleLevel
   asGrid
-  title="First level title*"
-  note="*Not to be used in content block (such as landing pages and login screens)"
+  note="Use only for extra large titles in cards or similar locations. Do not use for page titles"
+  styleProps={styleProps.superHero} />
+
+### Hero heading (3xl)
```

Comment:
The spacing seems a little off here. There is more space between the title and the section below that it pertains to than between the title and the previous section above.

<img width="745" alt="Screen Shot 2022-03-04 at 4 19 32 PM" src="https://user-images.githubusercontent.com/35148959/156851739-8a14de65-26d8-485f-9ca7-94d72ce3d1e8.png">

Looks like that's because `<TitleLevel>` expects the `title` property, and if you don't pass it, it creates an empty title element in the page - https://github.com/patternfly/patternfly-org/blob/main/packages/v4/src/content/design-guidelines/styles/typography/typography.js#L92

<img width="1238" alt="Screen Shot 2022-03-04 at 4 42 22 PM" src="https://user-images.githubusercontent.com/35148959/156851830-7f0bf717-1462-4419-b938-294a9ab23148.png">

If I'm reading the [<TitleLevel> component](https://github.com/patternfly/patternfly-org/blob/main/packages/v4/src/content/design-guidelines/styles/typography/typography.js#L92) code right, I think you can add `title` back and pass another prop `className="pf-c-title pf-m-xl"` and the title class/modifier can be used to control the text size. That's also dependent upon removing the custom font-size/weight CSS for `#usage-guidelines ~ h3.ws-title` as referenced above, unless you'd like to control that with CSS instead of title classes.